telematic_system/telematic_units/carma_cloud_bridge/cloud_nats_bridge/test/file_listener_test.py

```diff
@@ -48,17 +48,17 @@ def test_main(self):
         self.unit_setup(event_handler)
 
         #Verify new carma cloud message type is empty prior to adding TCR/TCM to the log file
-        assert (str(event_handler.getNewCarmaCloudMessageType()) == "")
+        assert (str(event_handler.findNewCarmaCloudMessage()) == 'None')
 
         #Append a TCR to the logfile and verify the new carma cloud message type updates appropriately
         self.append_TCR(newTCRLine)
         time.sleep(1)
-        assert (str(event_handler.getNewCarmaCloudMessageType()) == "TCR")
+        assert (str(event_handler.findNewCarmaCloudMessage()) == "None")
```

Comment:
Is there any reason we're looking for empty strings now instead of TCR and TCM?

Reply:
This function does not return TCR nor TCM

Reply:
updated